big_tests/tests/service_mongoose_system_stats_SUITE.erl

```diff
@@ -124,22 +134,32 @@ no_more_events_is_reported() ->
     Prev = get_events_collection_size(),
     timer:sleep(100),
     Post = get_events_collection_size(),
+    ct:log("No more events reported Prev:~p, Post:~p", [Prev,Post]),
```

Comment:
Is it a leftover or do we want to leave the log?

Reply:
removed